source/broadband/webpa_rbus.c

```diff
@@ -4,10 +4,15 @@
 #include <stdlib.h>
 #include <wdmp-c.h>
 #include <cimplog.h>
+#include <pthread.h>
 #include "webpa_rbus.h"
+#include "webpa_notification.h"
+
+#define CLOUD_CONN_ONLINE "cloud_conn_online_event"
```

Comment:
This may be need to be moved to webpa_rbus.h for unit testing. move if required during tests

Reply:
moved to webpa_rbus.h